USERS.md

```diff
@@ -159,6 +159,7 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [Nextdoor](https://nextdoor.com/)
 1. [Nikkei](https://www.nikkei.co.jp/nikkeiinfo/en/)
 1. [Nitro](https://gonitro.com)
+1. [CS IT Center, NYCU](https://it.cs.nycu.edu.tw)
```

Comment:
```suggestion
1. [NYCU, CS IT Center](https://it.cs.nycu.edu.tw)
```

Would this be alright? Just to make it fit the alphabetical order slightly better. :-)

Reply:
No problem, I'll push a new commit later!